script.js

```diff
@@ -1,12 +1,118 @@
 //You can edit ALL of the code here
+let tvShows = getAllShows() 
+ console.dir(tvShows)
+let  filteredEpisode ;
+ const header = document.getElementById("header");
+ let selectionBar = document.getElementById("selection-bar");
+ let selectionShowsBar = document.getElementById("selection-shows-bar")
+//const allEpisodes = getAllEpisodes();
+let allEpisodes 
+fetch(`https://api.tvmaze.com/shows/20/episodes`)
```

Comment:
show id "20", why not no 1. Just curiosity:)

Reply:
hi, It was just by accident hopefully will change it soon to all shows